ipu-plugin/images/Dockerfile

```diff
@@ -17,19 +17,29 @@ ENV P4_NAME=$P4_NAME
 
 COPY --from=builder /usr/src/ipu-opi-plugin/bin/ipuplugin /usr/bin/
 COPY ${P4_NAME}/${P4_NAME}.pkg /
-RUN mkdir -p /opt/p4/p4-cp-nws/bin/
+RUN mkdir -p /opt/p4/p4-cp-nws/bin/p4
 COPY bin/p4rt-ctl /opt/p4/p4-cp-nws/bin/
-#TODO: Update to newer package, according to release.
-COPY bin/p4runtime-2023.11.0-py3-none-any.whl /opt/p4/p4-cp-nws/bin/
+#TODO: Update to newer p4runtime, for latest release.
+COPY bin/p4runtime-2023.11.0/p4 /opt/p4/p4-cp-nws/bin/p4
 
 WORKDIR /
 LABEL io.k8s.display-name="IPU OPI Plugin"
 ENV PYTHONUNBUFFERED=1
-RUN apk add -U --no-cache python3 py3-pip'=='23.3.1-r0 \
+#Install deps for p4runtime(previously deps got installed per p4runtime wheel package's->p4runtime-2023.11.0.dist-info/METADATA)
+RUN apk add -U --no-cache python3'>='3.8  py3-pip'=='23.3.1-r0 \
     py3-grpcio'=='1.59.3-r0 \
     py3-protobuf'=='4.24.4-r0 \
     py3-netaddr'=='0.9.0-r0
-RUN python3 -m pip install --no-cache-dir --break-system-packages /opt/p4/p4-cp-nws/bin/p4runtime-2023.11.0-py3-none-any.whl
+
+#Install deps for p4runtime(previously deps got installed per p4runtime wheel package's->p4runtime-2023.11.0.dist-info/METADATA)
+RUN python3 -m pip install --no-cache-dir --break-system-packages -v "protobuf==4.25.0" "googleapis-common-protos==1.66.0"
+
+#Note: Redhat's build system does not allow p4runtime wheel package. Manually installing python protobuf
+#files(for p4runtime) under pip's installation path.
+RUN output=`pip show pip | grep Location` && \
```

Comment:
Nice way of finding the path! Way better than hardcoding to /usr/lib/python3.9/site-packages